### PR TITLE
strf: 6687 update stencil-paper

### PR DESCRIPTION
**Pull request by bc-williamkwon**
#### What?
STRF-6687
Change default behavior of {{getFontsCollection}} to use font-display: swap; for Google Fonts, allow configuration

#### Tickets / Documentation

Add links to any relevant tickets and documentation.

- [Link 1](https://jira.bigcommerce.com/browse/STRF-6687)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+<a name="1.15.10"></a>
+## <small>1.15.10 (2019-05-28)</small>
+
+* docs: update @bigcommerce/stencil-paper to 2.0.13
```
Comment (mattolson) [inline CHANGELOG.md]: Can you describe what paper 2.0.13 changes? And this should not be "docs" but rather "feat" (probably)

Comment (bc-williamkwon) [reply CHANGELOG.md]: fixed

Comment (bc-williamkwon) [reply CHANGELOG.md]: I put the number back down on package.json to 1.15.9.  Thinking 1.15.10 should encompass everything.

Comment (bc-williamkwon) [reply CHANGELOG.md]: I was told by Juned that there is a gulptask for generating changelog so deleted this.

Comment (junedkazi) [reply CHANGELOG.md]: No changelog for cli it is auto generated on release. Plz make sure your commit message describes the change in detail.

**Pull request merged**